docs/5.0/pages/database-access/faq.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Database Access FAQ
+description: Frequently asked questions about Teleport Database Access
+---
+
+# Database Access FAQ
```

Comment:
I would add answers to 

## Do you support X client? 
( go into details for what a client needs to work with Teleport )

## When will you support X database?
( How to contact us to request a new database )